pkg/nfd-master/nfd-master.go

```diff
@@ -753,3 +844,70 @@ func (m *nfdMaster) instanceAnnotation(name string) string {
 	}
 	return m.args.Instance + "." + name
 }
+
+// deleteTaints deletes node taint(s) and taints annotation
+func (m *nfdMaster) deleteTaints(c context.Context, r *pb.SetLabelsRequest) (*pb.SetLabelsReply, error) {
```

Comment:
nodename should be enough

```suggestion
func (m *nfdMaster) deleteTaints(c context.Context, nodeName string) (*pb.SetLabelsReply, error) {
```